dashboard/client/src/api/functionsApi.js

```diff
@@ -1,6 +1,8 @@
 import axios from 'axios';
 import moment from 'moment';
 
+const getRepoURL = annotations => annotations['com.openfaas.cloud.git-repo-url'] || '';
```

Comment:
From review call - could we return the hard-coded default from this method? Alternatively write a migration script and keep the code simple.

Reply:
I have sent you migration script and kept the code suimple